crates/ruff_linter/src/rules/flake8_pyi/snapshots/ruff_linter__rules__flake8_pyi__tests__PYI003_PYI003.pyi.snap

```diff
@@ -1,11 +1,10 @@
 ---
 source: crates/ruff_linter/src/rules/flake8_pyi/mod.rs
-snapshot_kind: text
 ---
 PYI003.pyi:4:4: PYI003 Unrecognized `sys.version_info` check
   |
 3 | if sys.version_info[0] == 2: ...
-4 | if sys.version_info[0] == True: ...  # Y003 Unrecognized sys.version_info check  # E712 comparison to True should be 'if cond is True:' or 'if cond:'
+4 | if sys.version_info[0] == True: ...  # Y003 Unrecognized sys.version_info check  # E712 comparison to True should be 'if cond is True...
```

Comment:
Hmm, using `...` is slightly problematic because it's a valid syntax element 😆 Any chance we can customize the trimming character, although I'm not sure what character to recommend: Is there a unicode equivalent of `...`?

Reply:
Yeah, `…` is `U+2026 HORIZONTAL ELLIPSIS`. Are you fine with that even though it renders very similarly to `...` (three separate periods)? Or do you want something entirely different?

But yes, we can change it. I'll do it in a way where I add a new API to `annotate-snippets` so that I can push the patch upstream too. (The `...` is currently hard-coded into `annotate-snippets`, so the simplest would be to just change what is hard-coded.)

Reply:
I think I'm fine with that. It's different enough to avoid confusion. It might still be slightly confusing if it happens to be right at the end of a clause header. We can also iterate on this based on user feedback.